google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -903,9 +938,14 @@ def format_extension_headers headers
         ##
         # Yielded to a block to accumulate changes for a patch request.
         class Updater < File
+          # @private Do not allow storage_class to be set in an update call.
```

Comment:
This is because `patch` doesn't support it? What if `patch` does support it in the future, would this be removed?